lib/send_sms.rb

```diff
@@ -0,0 +1,14 @@
+require 'twilio-ruby'
```

Comment:
It's good that you have this in a separate file. You could go further and wrap it all inside a new class.